Documentation/ExtensionArchitecture/FileStructure/_ComposerJson/_ExtendedComposer.json

```diff
@@ -4,7 +4,7 @@
     "description": "An example extension",
     "license": "GPL-2.0-or-later",
     "require": {
-        "php": "^7.4",
+        "php": "^7.4 || ^8.0",
```

Comment:
Why still allow 7.4?

Reply:
Because of 11.5 where 7.4 is the minimum PHP requirement (look at the TYPO3 constraint). When v13.0 comes out we will change this accordingly for main.